src/sizzle.js

```diff
@@ -147,22 +148,21 @@ var i,
 
 	// CSS escapes
 	// http://www.w3.org/TR/CSS21/syndata.html#escaped-characters
-	runescape = new RegExp( "\\\\([\\da-f]{1,6}" + whitespace + "?|(" + whitespace +
-		")|.)", "ig" ),
-	funescape = function( _, escaped, escapedWhitespace ) {
-		var high = "0x" + escaped - 0x10000;
-
-		// NaN means non-codepoint
-		// Support: Firefox<24
```

Comment:
Is that old Firefox workaround no longer needed considering that Sizzle doesn't drop old browsers?

Reply:
No, it's just that the new code doesn't encounter that bug because `high` is not used unless `nonHex` is falsy (i.e., we're replacing a nonempty hexadecimal escape).

Reply:
@gibson042 Interesting. Was the `NaN` check for `high` even needed before since `escapedWhitespace` was checked as a fallback anyway (as far as I understand the previous logic)?